tests/setup.js

```diff
@@ -18,6 +18,36 @@ const seedData = {
             source: "facebook.com/url_to_video",
         },
     ],
+    Surah: [
+        {
+            surah_id: 1,
+            surah_number: 1,
+            revelation_place: "Makkah",
+            name_complex: "",
```

Comment:
Let's not have empty strings in test seed data. You can get a name complex from https://api.quran.com/api/v4/chapters/2?language=en

You can also see https://quran.api-docs.io/v4/chapters/get-chapter for reference on where I got that data.